src/Features/CSharp/Portable/InlineDeclaration/CSharpInlineDeclarationCodeFixProvider.cs

```diff
@@ -39,11 +38,19 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
         }
 
         protected override async Task FixAllAsync(
-            Document document, ImmutableArray<Diagnostic> diagnostics, 
+            Document document, ImmutableArray<Diagnostic> diagnostics,
             SyntaxEditor editor, CancellationToken cancellationToken)
         {
             var options = await document.GetOptionsAsync(cancellationToken).ConfigureAwait(false);
 
+            // Gather all statements to be removed
+            // We need this to find the statements we can safely attach trivia to
+            var declarationsToRemove = new HashSet<StatementSyntax>();
+            foreach (var diagnostic in diagnostics)
+            {
+                declarationsToRemove.Add((LocalDeclarationStatementSyntax)diagnostic.AdditionalLocations[0].FindNode(cancellationToken).Parent.Parent);
```

Comment:
so what is the location in additionalLocations[0] and what findNode will return?

Reply:
if we are trying to find statement syntax node, how about just calling GetAncestor<StatementSyntax>().First() ?

Reply:
`AdditionalLocations[0]` contains the declarator as can be seen here https://github.com/dotnet/roslyn/pull/28793/files#diff-7634d845769381db613da4beabce8ef6R76 so `.Parent.Parent` should always give the correct `LocalDeclarationStatementSyntax` but `GetAncestor<LocalDeclarationStatementSyntax>()` should work as well (although slower)

Reply:
alright. just checking since it assumes span that map to the certain node to be in additional locations in the certain order which I am not sure whether that is hard contract or just happen to be that way.

if that assumption got broken, unit test will fail, so fine with that.